includes/Services/RemoteWikiFactory.php

```diff
@@ -369,6 +372,28 @@ public function unMarkExperimental(): void {
 		$this->newRows['wiki_experimental'] = 0;
 	}
 
+	public function getExtraFieldData( string $field ): mixed {
+		return $this->extra[$field] ?? null;
+	}
+
+	public function setExtraFieldData( string $field, mixed $value ): void {
+		if ( $value !== $this->getExtraFieldData( $field ) ) {
```

Comment:
[Early return](https://gomakethings.com/the-early-return-pattern-in-javascript/)?

Reply:
Typically I like early returns but decided not to here as my is to eventually make RemoteWikiFactory more similar to how WikiRequestManager works which means it being more inline with that here makes it a tiny bit easier down the line which is why I decided against it. But I guess it wouldn't be horrible to early return also...

Reply:
Alright then, feel free to resolve this conversation if you decide not to early return